### PR TITLE
small typo change to solve the issue 1930

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
A small change in the desc of the operator-sdk build . 

**Motivation for the change:**

- https://github.com/operator-framework/operator-sdk/issues/1930

### CODE REVIEW

File: cmd/operator-sdk/build/cmd.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func NewCmd() *cobra.Command {
 		Use:   "build <image>",
 		Short: "Compiles code and builds artifacts",
 		Long: `The operator-sdk build command compiles the code, builds the executables,
```
Comment (estroz) [inline cmd/operator-sdk/build/cmd.go]:
```suggestion
		Long: `The operator-sdk build command compiles the Operator code into an executable binary
```

File: cmd/operator-sdk/build/cmd.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func NewCmd() *cobra.Command {
...
-and generates Kubernetes manifests.
+and generates the Dockerfile manifest.
```
Comment (estroz) [inline cmd/operator-sdk/build/cmd.go]:
```suggestion
and builds a Docker image with the specified image name.
```

Comment (camilamacedo86) [reply cmd/operator-sdk/build/cmd.go]: Indeed it is better 👍

File: cmd/operator-sdk/build/cmd.go (hunk 1 above)
Comment (camilamacedo86) [reply cmd/operator-sdk/build/cmd.go]: 👍

**Pull request merged**